cartographer/mapping/internal/2d/pose_graph/optimization_problem_2d.cc

```diff
@@ -273,12 +273,27 @@ void OptimizationProblem2D::Solve(
         continue;
       }
 
-      const transform::Rigid3d relative_pose =
-          ComputeRelativePose(trajectory_id, first_node_data, second_node_data);
+      // Add a relative pose constraint based on the odometry (if available).
+      std::unique_ptr<transform::Rigid3d> relative_odometry =
+          OdometryBetween(trajectory_id, first_node_data, second_node_data);
```

Comment:
usually we try to start function names with a verb unless they are simple accessors. How about `CalculateOdometryBetweenNodes()`?

Reply:
> CalculateOdometryBetweenNodes()

That's what we do, sounds good.